Week1/modulo.js

```diff
@@ -0,0 +1,10 @@
+let m = 7;
+m = m%3;
+console.log("When we divide 7 by 3, reminder is "+ m);
+let r = 458;
+let p = 23;
+let modulo = r%p;
+console.log("when you divide the number "+r+" by the number "+p+" the reminder is the number "+modulo);
+let two = 2;
+let reminder=r%two;
```

Comment:
I would suggest adding some spaces to make the code more readable:

`let reminder = r % two;`